src/renderer/pages/index.vue

```diff
@@ -810,7 +812,9 @@ export default Vue.extend({
   }
 
   .accent *,
-  .success * {
+  .success *, 
+  .warning *, 
+  .info * {
```

Comment:
Show-hide media window button doesn't look nice in dark mode

<details><summary>See image</summary>

![Screenshot from 2022-10-04 09-55-49](https://user-images.githubusercontent.com/46671786/193766362-48b311ca-b3b9-41b0-ba87-7ea24144c715.png)

</details>